src/res/translation/translation_fr_FR.ts

```diff
@@ -840,7 +840,7 @@
     <message>
         <location filename="../../clientdlg.cpp" line="319"/>
         <source>Auto-Adjust all &amp;Faders</source>
-        <translation type="unfinished"></translation>
+        <translation>Auto-ajuster tous les chariots &amp;f</translation>
```

Comment:
Hm, what's the reason for the final `&amp;f`? Just the accelerator key?
I think accelerator keys have been language-specific in the past -- with all the up and downsides (@softins  ;)), so it may be better to choose a character which matches a relevant french word as well?

Reply:
I did wonder how I should handle the accelerator keys. the final `&amp;f` is only for that accelerator to be the same as in english. I think we should have a discussion around this. :-)

Reply:
My impression is that accelerator keys are often the first character of the first (or most relevant) word. If this is not possible due to conflicts, alternatives have to be found. I think it makes sense to try to align these across languages, especially if the words are similar like for several languages with the same roots.
However, if the English accelerator key has no occurence in the translated string, I think an alternative should be found.

Maybe @softins or other translators can comment regarding this as well.

You may also be interested in the output of this tool by @softins:
```
jamulus/src/res/translation $ ../../../tools/checkkeys.pl
[...]
Possible duplicate hotkeys:

de_DE
	CChatDlg
		Key: E
			B&earbeiten (&Edit)
			Lösch&en des Chatverlaufs (Cl&ear Chat History)
	CClientDlg+CHelpMenu
		Key: B
			&Beenden (E&xit)
			Ü&ber... (&About...)
		Key: E
			B&earbeiten (&Edit)
			&Erste Schritte... (Getting &Started...)
```

(This is just a helper which shows the cases which have to be checked manually; I'm just referencing it as it provides a partial overview over such keys)